### PR TITLE
Update factory calendar

### PR DESCRIPTION
**Pull request by mervey45**

### CODE REVIEW

**Comment by thecodester** [comment]
@mervey45 Will the new tutorial be part of a group? Also, add a little more to the description, and mention ABAP environment or the technology involved.

**Comment by mervey45** [comment]
Hi @thecodester can you merge my changes?

Thank you.

Best regards, 
Merve

File: tutorials/abap-environment-text-table/abap-environment-text-table.md [hunk 1]
```diff
@@ -279,59 +310,66 @@ Many business configuration objects use text tables for providing translatable a
 
   2. Save and activate.
 
-  3. Let's now continue and directly add the annotations for table display. Open your text table CDS view **`ZCAL_I_HOLITXT_XXX`**.
+  3.  Open **`ZCAL_I_HOLITXT_XXX`** and add line times
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-text-table/abap-environment-text-table.md:313]: line times - Is this a typo and should be "line items"?

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/abap-environment-text-table/abap-environment-text-table.md (hunk 1 above)
Comment (mervey45) [reply tutorials/abap-environment-text-table/abap-environment-text-table.md:313]: Yes, it is. Should I change it?

Comment (MichaelCzcz) [inline tutorials/abap-environment-text-table/abap-environment-text-table.md:313]:
```suggestion
  3.  Open **`ZCAL_I_HOLITXT_XXX`** and add line items
```

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Pull request merged**